guides/v2.2/migration/migration-tool-internal-spec.md

```diff
@@ -108,7 +108,7 @@ Script that runs migration process is located at magento-root/bin/magento
 
 The Schema for configuration file `config.xsd` is placed under `etc/directory`. Default configuration file `config.xml.dist` is created for each version of Magento 1.x. It is placed in separate directories under `etc/`.
 
-Default configuration file can be replaced by custom one using CLI (see [`--config <value>` parameter]({{ page.baseurl }}/migration/migration-migrate.html)).
+Default configuration file can be replaced by custom one using CLI (see [`--config <value>` parameter]({{ page.baseurl }}/migration/migration-migrate.html#migration-command-run-syntax)).
```

Comment:
Maybe we should remove the `[`--config <value>` parameter]` part. If you check https://devdocs.magento.com/guides/v2.3/migration/migration-migrate.html#migration-command-run-syntax, path to the custom config file is passed without specifying an argument name (`--config`)

Reply:
Agree. Updated. Thank you